pyplanet/apps/contrib/jukebox/__init__.py

```diff
@@ -45,7 +45,8 @@ def __init__(self, *args, **kwargs):
 			Command(command='clearjukebox', target=self.clear_jukebox, perms='jukebox:clear', admin=True),
 			Command(command='list', target=self.show_map_list).add_param(name='search', required=False),
 			Command(command='jukebox', target=self.chat_command).add_param(name='option', required=False),
-			Command(command='mapfolders', target=self.show_map_folders)
+			Command(command='mapfolders', target=self.show_map_folders),
+			Command(command='mf', target=self.show_map_folders)
```

Comment:
The Command class has an aliases argument you can supply when initializing it, so I dont think it's needed to add a new command for this. `Command(command='mapfolders', aliases=['mf'] ... ` should work too. Not sure if this will show up in the commands list though when doing /help

Reply:
@TheSuperNoob Thanks for your contribution, the code has been refactored